### PR TITLE
Fix up Raptor Configs With Available Info on them

### PR DESCRIPTION
**Pull request by vader111**
Sources have been updated.

Some values are semi-guesstimates.

### CODE REVIEW

File: GameData/RealismOverhaul/Engine_Configs/Raptor.cfg [hunk 1]
```diff
@@ -7,40 +7,42 @@
...
-//	Chamber Pressure: 27 MPa
+//	Chamber Pressure: 33 MPa
 //	Propellant: LOX / CH4
-//	Prop Ratio: 3.8
-//	Throttle: 20% - 100%
-//	Nozzle Ratio: 161
-//	Ignitions: ???
+//	Prop Ratio: 3.55
+//	Throttle: 40% - 100%
+//	Nozzle Ratio: 107 ~ Not Sure if its the same as expansion ratio?
```
Comment (Standecco) [inline GameData/RealismOverhaul/Engine_Configs/Raptor.cfg:34]: Should be the same thing, yeah. It's the ratio between the nozzle's exit area and its throat section.

File: GameData/RealismOverhaul/Engine_Configs/Raptor.cfg [hunk 2]
```diff
@@ -52,7 +54,7 @@
 {
 	%title = Raptor
 	%manufacturer = SpaceX
-	%description = The Raptor is an full flow staged combustion engine that burns CH4/LOX. Designed for the Interplanetary Transport System, 42 sea level optimised engines power the first stage. The Mars ship has 3 sea level engines and 6 vacuum engines.
+	%description = The Raptor is an CH4/LOX burning full flow staged combustion engine designed for both the Starship & Super Heavy Launch Vehicle. The Super Heavy has 31 sea level optimised engines to power the first stage, Starship has 3 sea level engines and 3 vacuum engines.
```
Comment (Standecco) [inline GameData/RealismOverhaul/Engine_Configs/Raptor.cfg:57]:
```suggestion
	%description = The Raptor is a CH4/LOX burning full-flow staged combustion engine designed for both the Starship & Super Heavy Launch Vehicle. The Super Heavy has 31 sea-level optimized engines to power the first stage, Starship has 3 sea-level engines and 3 vacuum engines.
```
Minor typos.

**Comment by siimav** [comment]
> Raptor Superheavy - **330 SL** / 355 Vac Isp
Raptor Starship - **350 SL** / 380 Vac Isp

This doesn't make any sense

File: GameData/RealismOverhaul/Engine_Configs/Raptor.cfg [hunk 3]
```diff
@@ -7,40 +7,42 @@
...
 //	Starship
 //
-//	Dry Mass: ??? Kg
-//	Thrust (SL): ??? kN
-//	Thrust (Vac): 1957.22 kN
-//	ISP: ??? SL / 379 Vac
+//	Dry Mass: 200 TWR Ratio ~ Goal
+//	Thrust (SL): ???
+//	Thrust (Vac): 2241.90 kN ~ Guess
+//	ISP: 350 SL / 380 Vac
```
Comment (Standecco) [inline GameData/RealismOverhaul/Engine_Configs/Raptor.cfg:28]: Seems very unlikely to me that the Vacuum optimized version would have higher sea-level Isp than the SL optimized version. Should probably be lower, the Raptor barely avoids flow separation. I'd look at the SSME and other similar sustainer engines and check how much their SL specific impulses vary with pressure.

File: GameData/RealismOverhaul/Engine_Configs/Raptor.cfg [hunk 4]
```diff
@@ -76,74 +78,125 @@
...
+			%pressureFed = False
+			%ignitions = 0
+
 			IGNITOR_RESOURCE
 			{
 				name = ElectricCharge
 				amount = 0.5
 			}
-		
+			IGNITOR_RESOURCE // Touch-Lit System uses the same propellant has the engines
```
Comment (Standecco) [inline GameData/RealismOverhaul/Engine_Configs/Raptor.cfg:114]:
```suggestion
			IGNITOR_RESOURCE // Torch-Lit System uses the same propellant has the engines
```
Small typo, also present in other parts of the config. I won't highlight all of them, but there are 7 other occurrences

**Review by Standecco — CHANGES_REQUESTED** [review]
Looks good, but the SL Isp of the Vacuum optimized variant seems unlikely. Read comment for more info

File: GameData/RealismOverhaul/Engine_Configs/Raptor.cfg (hunk 3 above)
Comment (vader111) [reply GameData/RealismOverhaul/Engine_Configs/Raptor.cfg:28]: What would you guess would be a good surface isp for the vacuum engines?

**Comment by vader111** [comment]
Should be good now

**Comment by DRVeyl** [comment]
Any further comments on the stats/sourcing/etc?

**Comment by vader111** [comment]
@DRVeyl If more stats and come to light about the raptor engines then i will update the config as there come out

**Comment by Capkirk123** [comment]
Everything looks reasonable. What's the non-throttleable one for?

**Comment by vader111** [comment]
> 
> 
> Everything looks reasonable. What's the non-throttleable one for?

https://youtu.be/-8p2JDTd13k?t=1085

Its more powerful Raptor that will be used on the outer ring of the superheavy booster

**Pull request merged**